analytics/lib/counts.py

```diff
@@ -326,5 +326,9 @@ def do_pull_from_zerver(stat, start_time, end_time, interval):
                                          (UserProfile, 'is_bot'), CountStat.DAY, False),
     'message_sent_by_type_by_user': CountStat('messages_sent_by_type_by_user',
                                               zerver_count_message_type_by_user, {},
-                                              None, CountStat.DAY, False)
-}
+                                              None, CountStat.DAY, False),
+    'messages_sent_by_client': CountStat('messages_by_client', zerver_count_message_by_user, {},
+                                         (Message, 'sending_client_id'), CountStat.HOUR, False),
+    'messages_sent_by_is_bot_by_stream': CountStat('messages_by_humans_by_stream', zerver_count_message_by_stream,
+                                              {'is_active': True}, (UserProfile, 'is_bot'), CountStat.HOUR, False)
+    }
```

Comment:
Let's change all these names (and properties) to be a little less unwieldy. The names should = the property. How about
active_users:is_bot
messages_sent
messages_sent:is_bot
messages_sent:stream_type
messages_sent:client
messages_sent_by_stream:is_bot

Could you change the first four in a separate commit, and then add this commit on top of that?